sw/device/lib/testing/test_framework/ottf_isrs.h

```diff
@@ -137,6 +137,33 @@ void ottf_timer_isr(uint32_t *exc_info);
  */
 void ottf_external_isr(uint32_t *exc_info);
 
+/**
+ * Test external IRQ handler.
+ *
+ * `ottf_isrs.c` provides a weak definition of this symbol, which can be
+ * overridden at link-time by providing an additional non-weak definition.
+ *
+ * Overriding this function is prefered way for device tests to handle
+ * IRQ unless they really need to bypass the OTTF entirely.
+ *
+ * Before calling this function, `ottf_external_isr` will claim the interrupt
+ * at the PLIC and obtain the corresponding instance ID using the DT.
+ * If this function returns true, `ottf_external_isr` will complete the
+ * interrupt at the PLIC. If this function returns false, the OTTF will try
+ * to handle this IRQ for its internal functions (e.g. ottf_console). If that's
+ * not possible, a fatal error will occur.
+ *
+ * NOTE When overriding `ottf_handle_irq`, the code does not need to call
+ * `ottf_console_flow_control_isr`, this will be done automatically if the
```

Comment:
```suggestion
 * NOTE When overriding `ottf_handle_irq` the code does not need to call
 * `ottf_console_flow_control_isr`: this will be done automatically if the
```